packages/react/test/specs/components/Dropdown/Dropdown-test.tsx

```diff
@@ -34,20 +35,169 @@ describe('Dropdown', () => {
     })
   })
 
+  describe('open', () => {
+    const onOpenChange = jest.fn()
+
+    afterEach(() => {
+      onOpenChange.mockReset()
+    })
+
+    it('is false when closed by trigger button click', () => {
+      const wrapper = mountWithProvider(<Dropdown onOpenChange={onOpenChange} items={items} />)
+      const triggerButton = wrapper.find(`button.${Dropdown.slotClassNames.triggerButton}`)
+
+      triggerButton.simulate('click').simulate('click')
+
+      expect(onOpenChange).toBeCalledTimes(2)
+      expect(onOpenChange).toHaveBeenLastCalledWith(
+        null,
+        expect.objectContaining({
+          open: false,
+        }),
+      )
+    })
+
+    it('is false when closed by hitting Escape in search input', () => {
+      const wrapper = mountWithProvider(
+        <Dropdown onOpenChange={onOpenChange} search items={items} />,
+      )
+      const searchInput = wrapper.find(`input.${DropdownSearchInput.slotClassNames.input}`)
+
+      searchInput
+        .simulate('click')
+        .simulate('change', { target: { value: 'test' } })
+        .simulate('keydown', { keyCode: keyboardKey.Escape, key: 'Escape' })
+
+      expect(onOpenChange).toBeCalledTimes(2)
+      expect(onOpenChange).toHaveBeenLastCalledWith(
+        null,
+        expect.objectContaining({
+          open: false,
+        }),
+      )
+    })
+
+    it('is false when closed by hitting Escape in items list', () => {
+      const wrapper = mountWithProvider(
+        <Dropdown onOpenChange={onOpenChange} multiple items={items} />,
+      )
+      const triggerButton = wrapper.find(`button.${Dropdown.slotClassNames.triggerButton}`)
+
+      triggerButton
+        .simulate('click')
```

Comment:
Shouldn't we first navigate to the list items by arrow down and then press escape? Otherwise we are laying about what the test is chaeking..

Reply:
we are already checking that on trigger click, the focus goes to the list, it's a test called `'opens the menu and moves focus to list in selection mode'`

so assuming this works, the focus is already in the list, and we hit Escape to close it.